baselines/EMNLP2019/README.md

```diff
@@ -0,0 +1,62 @@
+## Requirements
+python 3.7
+```
+pip install -r requirements.txt
+python -m spacy download en_core_web_lg
+```
+
+Set environment variable DIFFBOT_TOKEN if you want to use entitylinking. We are providing cached results for the KnowledgeNet documents, but you will need this if you want to run the system on other documents or if you want to change the NER system. Free trial available at https://www.diffbot.com/get-started/
```

Comment:
I think it's better to replace "Free trial available at https://www.diffbot.com/get-started/" by "Contact Filipe Mesquita (filipe[at]diffbot.com) for a free research token."